cmake/onnxruntime_unittests.cmake

```diff
@@ -564,8 +564,8 @@ if(NOT TARGET onnxruntime)
 endif()
 
 if (onnxruntime_USE_CUDA)
-  add_library(onnxruntime_test_cuda_add_lib ${ONNXRUNTIME_SHARED_LIB_TEST_SRC_DIR}/cuda_add.cu)
-  list(APPEND onnxruntime_test_common_libs onnxruntime_test_cuda_add_lib)
+  add_library(onnxruntime_test_cuda_ops_lib ${ONNXRUNTIME_SHARED_LIB_TEST_SRC_DIR}/cuda_ops.cu)
+  list(APPEND onnxruntime_test_common_libs onnxruntime_test_cuda_ops_lib)
```

Comment:
Was this just due to a rename of cuda_add.cu to cuda_ops.cu?

Reply:
Yes, just because of renaming.